src/main/java/com/epam/prd/jjop/helpers/objects/NullWonder.java

```diff
@@ -0,0 +1,79 @@
+package com.epam.prd.jjop.helpers.objects;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.Objects;
+
+/**
+ * Description of @see java.util.Objects regarding null problem solving
+ * <p>
+ * This class show examples of all methods to check for null.
+ * Description of the differences between these methods can be found here.
+ * </p>
+ *
+ * @author Krzysztof Dolański
+ * Objects class form java.util @since 1.7
+ */
+
+public class NullWonder {
+
+
+    public static void main(String[] args) {
+
+        String assumeNull = null;
+
+        /**
+         * <p>
+         * Using null object as parameter in this method will produce 0 as return.
+         * </p>
+         * <p>
+         * This method under the hood is using Arrays.hashCode
+         * </p>
+         * <p>
+         * WARNING - in case of passing more than one null parameter return value WILL NOT be 0 because of implementation of Arrays.hashCode()
```

Comment:
```suggestion
         * WARNING - in case of passing more than one null parameter return value WILL NOT be 0 because of implementation of {@link java.util.Arrays.hashCode()}.
```